docs/features.md

```diff
@@ -0,0 +1,25 @@
+# Features
+
+Some of the key features of Percona Everest are:
+
+- **Multi-Cloud and on-premise deployments**: Percona Everest enables you to transcend the limitations of a single cloud provider. Deploy your databases across multiple clouds or on-premises infrastructure, including OpenShift, to ensure redundancy, high availability, and data sovereignty.
+- **Multiple open source database support**: Percona Everest offers managed solutions for popular open-source databases, including PostgreSQL, MySQL, and MongoDB.
+- **Private deployments and DIY**: Percona Everest provides complete control over your database environment, ensuring the privacy and security of your data. Private deployments enable you to customize every aspect of your database setup using a do-it-yourself approach.
+- **Database provisioning**
+    * **Horizontal and vertical scaling**: Percona Everest enables you to adapt to your application's demands using horizontal scaling for multi-node deployments or vertical scaling for single-node setups. With Percona Everest, you have the power to manage resources and ensure your databases are performing at their best.
+    * **Database storage class support**: Percona Everest tailors your storage needs efficiently by leveraging its database storage class support. Furthermore, it allocates resources efficiently while maintaining optimal performance and cost.    
+    * **Disaster recovery capabilities**: Percona Everest prioritizes data protection with a comprehensive disaster recovery suite. Set up backup schedules, create on-demand backups, perform point-in-time recovery, seamlessly restore existing databases, or create new ones from backups.
+    * **Advanced configuration options**: Percona Everest gives fine-grained control of your database environment with advanced configuration features. You can manage external access permissions and optimize your database engine configuration.
+
+- Database Management Features
+
+    * **Effortless Updates**: With Percona Everest, you can stay up-to-date with the latest features and security patches. Percona Everest is built with Kubernetes Operators that help simplify database updates by automating the process and doing it without downtime, ensuring your databases are proofed against vulnerabilities.
+
+    * **Resource Allocation Flexibility**: You can handle changing workloads by adjusting your resource allocations on the fly. Whether you need more processing power or memory, Percona Everest's vertical scaling has you covered.
+
+    * **Backup Management**: You can take on-demand backup and restore the data to ensure that your data is protected.
+
+- **Administrative Features**
+
+    * **Centralized Configuration Management**: Percona Everest simplifies administrative tasks by allowing you to add and manage storage locations, monitoring endpoints, storage classes, and other default configurations from a central interface.
```

Comment:
we don't have storage management or defaults yet

Reply:
Removed